### DISCUSSION TITLE
theme "Aura" not found

### DISCUSSION THREAD

**Discussion by lukas-cc** [ghostty-org/ghostty#3701] [Q&A] [answered]
I get this error when trying to set `theme = Aura`. However, it is listed in [iterm2colorschemes](https://iterm2colorschemes.com/). Any idea what might be the issue here ?

**Comment by 00-kat** [comment, marked as answer]
Merged 15 hours ago https://github.com/mbadolato/iTerm2-Color-Schemes/pull/475.  Give it some time (assuming you're on the tip build/building from source without using a tag), Ghostty has a specific hash it clones and it's not got around to grabbing that yet.

For now, you can grab the file yourself from the repository https://github.com/mbadolato/iTerm2-Color-Schemes/blob/master/ghostty/Aura.

**Reply by lukas-cc** [reply]
Great thanks!
